insertBaseData-mssql.sql

```diff
@@ -422,24 +422,24 @@ INSERT INTO approvers (approvalStepUuid, positionUuid)
 	AND approvalSteps.type = 1;
 
 -- Create locations
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'cc49bb27-4d8f-47a8-a9ee-af2b68b992ac', 'St Johns Airport', 47.613442, -52.740936, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'8c138750-91ce-41bf-9b4c-9f0ddc73608b', 'Murray''s Hotel', 47.561517, -52.708760, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'9c982685-5946-4dad-a7ee-0f5a12f5e170', 'Wishingwells Park', 47.560040, -52.736962, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'0855fb0a-995e-4a79-a132-4024ee2983ff', 'General Hospital', 47.571772, -52.741935, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'95446f93-249b-4aa9-b98a-7bd2c4680718', 'Portugal Cove Ferry Terminal', 47.626718, -52.857241, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'c8fdb53f-6f93-46fc-b0fa-f005c7b49667', 'Cabot Tower', 47.570010, -52.681770, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'c7a9f420-457a-490c-a810-b504c022cf1e', 'Fort Amherst', 47.563763, -52.680590, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'7339f9e3-99d1-497a-9e3b-1269c4c287fe', 'Harbour Grace Police Station', 47.705133, -53.214422, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
-INSERT INTO locations (uuid, name, lat, lng, createdAt, updatedAt)
-	VALUES (N'f2207d9b-204b-4cb5-874d-3fe6bc6f8acd', 'Conception Bay South Police Station', 47.526784, -52.954739, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+INSERT INTO locations (uuid, name, lat, lng, gisPoint, createdAt, updatedAt)
```

Comment:
we need to remove the lat and lng columns and replace them with a single column for GIS geometry (as we have gisPoint now, but probably named gisGeometry as we may not need to be contrained to a point)

Reply:
Removing `lat` and `lng` columns;

- will result in substantial amount of changes in both backend and frontend.
- will result in conflicting changes (potentially hard to resolve) with our currently open PRs #3073 #3062 and #3056.
- will block progress on #3044.
- will not provide any new functionality.

Due to above reasons I suggest keeping this PR as is and open a debt issue for removing `lat` and `lng` columns. Also postpone that issue until #3073 #3062 and #3056 are merged and #3044 is implemented.

Reply:
Changed column name to _geometry_

Reply:
The issue is that by merging this PR, we will have the geo data twice in the database - once as lat/lon and once as a geometry. Moreover, when new location objects are created or the ones modified, their geomerty is not updated, so the two do get out of sync. Even if they were in sync, it would be combersome to manipulate the database if needed with external scripts.